ui/app/src/components/Navigation/PathNavigator/PathNavigatorSkeletonItem.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { useStyles } from './styles';
+import ListItem from '@material-ui/core/ListItem';
+import { Skeleton } from '@material-ui/lab';
```

Comment:
Check all lab imports to be loaded from its individual file (e.g. material/lab/skeleton)